parastell/parastell.py

```diff
@@ -349,19 +354,45 @@ def _tag_materials_native(self):
                 vol_id_str = str(block_id)
                 make_material_block(data['mat_tag'], block_id, vol_id_str)
 
-    def export_dagmc(
-        self, skip_imprint=False, legacy_faceting=True, filename='dagmc',
-        export_dir='', **kwargs
-    ):
-        """Exports DAGMC neutronics H5M file of ParaStell components via
-        Coreform Cubit.
+    def build_cubit_model(
+            self, skip_imprint=False, legacy_faceting=True, **kwargs
+            ):
```

Comment:
Maintain proper indentation.
```suggestion
    def build_cubit_model(
        self, skip_imprint=False, legacy_faceting=True, **kwargs
    ):
```